src/app/widgets/reactive-fields/reactive-label/reactive-label.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ReactiveLabelComponent } from './reactive-label.component';
+
+@NgModule({
+  imports: [CommonModule],
+  exports: [ReactiveLabelComponent],
```

Comment:
Ye. We export it here.

Reply:
Yes, you're right. I deleted the export `ReactiveLabelComponent` from `index.ts`